test/integration/authorizations_test.go

```diff
@@ -27,169 +27,60 @@ const envKeyClientSecret = "GITHUB_CLIENT_SECRET"
-		t.Fatal("The returned Authorization does not match the requested updated value.")
-	}
-
-	// Verify that the Get operation also reflects the update
-	retrievedAuth, resp, err := client.Authorizations.Get(context.Background(), *createdAuth.ID)
-	failOnError(t, err)
-	failIfNotStatusCode(t, resp, 200)
-
-	if *retrievedAuth.Note != *updatedAuth.Note {
-		t.Fatal("The retrieved Authorization does not match the expected (updated) value.")
-	}
-
-	// Now, let's delete...
-	resp, err = client.Authorizations.Delete(context.Background(), *createdAuth.ID)
-	failOnError(t, err)
-	failIfNotStatusCode(t, resp, 204)
-
-	// Verify that we can no longer retrieve the auth
-	retrievedAuth, resp, err = client.Authorizations.Get(context.Background(), *createdAuth.ID)
-	if err == nil {
-		t.Fatal("Should have failed due to 404")
-	}
-	failIfNotStatusCode(t, resp, 404)
-
-	// Verify that our count reset back to the initial value
-	auths, resp, err = client.Authorizations.List(context.Background(), nil)
-	failOnError(t, err)
-	failIfNotStatusCode(t, resp, 200)
-
-	if len(auths) != initialAuthCount {
-		t.Fatalf("The number of Authorizations should match the initial count Expected [%v], got [%v]", initialAuthCount, len(auths))
-	}
-
-}
-
 // TestAuthorizationsAppOperations tests the application/token related operations, such
 // as creating, testing, resetting and revoking application OAuth tokens.
 func TestAuthorizationsAppOperations(t *testing.T) {
 
-	userAuthenticatedClient := getUserPassClient(t)
-
 	appAuthenticatedClient := getOAuthAppClient(t)
 
 	// We know these vars are set because getOAuthAppClient would have
 	// skipped the test by now
 	clientID := os.Getenv(envKeyClientID)
-	clientSecret := os.Getenv(envKeyClientSecret)
 	accessToken := os.Getenv(envKeyAccessToken)
 
-	authRequest := generateAppAuthTokenRequest(clientID, clientSecret)
-
-	createdAuth, resp, err := userAuthenticatedClient.Authorizations.GetOrCreateForApp(context.Background(), clientID, authRequest)
-	failOnError(t, err)
-	failIfNotStatusCode(t, resp, 201)
-
-	// Quick sanity check:
-	if *createdAuth.Note != *authRequest.Note {
-		t.Fatal("The returned auth does not match expected value.")
-	}
-
-	// Let's try the same request again, this time it should return the same
-	// auth instead of creating a new one
-	secondAuth, resp, err := userAuthenticatedClient.Authorizations.GetOrCreateForApp(context.Background(), clientID, authRequest)
-	failOnError(t, err)
-	failIfNotStatusCode(t, resp, 200)
-
-	// Verify that the IDs are the same
-	if *createdAuth.ID != *secondAuth.ID {
-		t.Fatalf("The ID of the second returned auth should be the same as the first. Expected [%v], got [%v]", createdAuth.ID, secondAuth.ID)
-	}
-
 	// Verify the token
-	appAuth, resp, err := appAuthenticatedClient.Authorizations.Check(context.Background(), clientID, *createdAuth.Token, accessToken)
+	_, resp, err := appAuthenticatedClient.Authorizations.Check(context.Background(), clientID, accessToken)
```

Comment:
Why are all the "Quick sanity checks" being removed from the integrations tests?

Reply:
`GetOrCreateForApp`. This function uses deprecated git api `PUT /authorizations/clients/:client_id/:fingerprint`. So had to remove everything which is dependent on it.

Reply:
My point is that `Authorizations.Check` still exists even though it is using a new API... so this integration should now use the new returned values and perform an equivalent "Quick sanity check" like it was doing before. Isn't that possible?